src/Elliptic/Systems/Xcts/Equations.cpp

```diff
@@ -61,60 +61,61 @@ void add_hamiltonian_sources(
     const gsl::not_null<Scalar<DataVector>*> hamiltonian_constraint,
```

Comment:
[musing] It might be worth revisiting this in a followup performance review of the elliptic code. It might be possible to precompute the conformal factor and then pass it around. I think this is low priority, but if you have a list somewhere to keep track of these things, maybe add this?

Reply:
Yes, will do. I have no idea if precomputing it is faster or not.